src/legacy/core_plugins/console/np_ready/public/legacy.ts

```diff
@@ -21,6 +21,7 @@ import 'brace';
 import 'brace/ext/language_tools';
 import 'brace/ext/searchbox';
 import 'brace/mode/json';
+import 'brace/mode/sql';
```

Comment:
Is this needed? I tried removing it and the syntax highlighting looks fine.

Reply:
Will remove!